content/en/seachat/seachat-manual/02-create-agent/04-seachat-conversations.md

```diff
@@ -44,3 +44,53 @@ In each conversation row, there will be an icon indicating the conversation chan
+
+## Download Agent Conversation History
+
+If you wish to download the conversation history of certain agent conversations, you can do so by navigating to **Workspace*  - `-> **Agents**. 
+
+Click on **Download Conversations*  - `and Choose the year of the conversation history you wish to download. 
+
+<br/>
+<center>
+<a style="border-radius: 0.4rem; cursor: zoom-in;" href="/images/seachat/en/seachat-conversation/download-conversation-history.png" target="_blank">
+<img width="80%" style="border-radius: 0.4rem" src="/images/seachat/en/seachat-conversation/download-conversation-history.png" alt="">
+</a>
+
+**Download Conversation History**
+</center>
+
+This process might take a while depending on the size of the conversation history. Once the download is ready, you will receive a system notification says `AI agent conversation history export successful. Please click link or copy the link to browser to download. The link will expire in 24 hours.`
+
+Click on the link to download the conversation history in a zip file format. 
+
+You will see the conversation history in a CSV file format. The CSV file contains the following columns:
+
+| Sender type | Channel type | Sender name | Time in GMT | Message | Data |
+|-------------|--------------|-------------|-------------|---------|------|
+
+- **Sender type**: The type of sender. You can identify whether the sender is the agent, system notification, user, etc.
+- **Channel type**: The communication channel of the conversation. The following is the list of channel types:
+   - `System (this means system message)`
+   - `WebChat`
+   - `SeaAuth (this is usually for verification purpose)`
+   - `Line`
+   - `WhatsApp`
+   - `Messenger`
+   - `Instagram`
+   - `Voice (this means phone calls)`
+   - `ThirdPartyClient`
+   - `AgentPhone (this means phone calls from SeaChat voice agent, usually for token verification purposes)`
+   - `SeaX_Call (this means phone calls through SeaX)`
+   - `SeaX_SMS (this means SMS through SeaX)`
+- **Sender name**: The name of the sender. You will find the senders' ID or Email in this column if the sender is an actual user. Otherwise, it will be an identifier from the system.
+- **Time in GMT**: The timestamp of the message in GMT format.
+- **Message**: The message content of the conversation. The following is the list of message types:
+   - `/live_agent:  User requests conversation transfer to a live agent.`
+   - `/submit_form: User submits a form. If an entity is involved, the command will appear as /submit_form{"entity_name":"entity_value"}.`
+   - `/clear_history: User requests the bot to forget the conversation history and start fresh.`
+   - `/ai_agent: User requests conversation transfer to an AI agent.`
+- **Data**: The data content of the conversation in JSON format. This column will be empty if there is no data content in the conversation. The following is the list of data types:
```

Comment:
- **Data**: The data content of the conversation in JSON format. This column will be empty if there is no data content in the conversation. When data is present, it can include, but is not limited to, the following types:
...